layouts/README.txt

```diff
@@ -0,0 +1,13 @@
+Place downloaded and custom layouts in this directory to ensure clean separation
+from Backdrop core and to facilitate safe, self-contained code updates.
+Contributed layouts from the Backdrop community may be downloaded at
+http://drupal.org/project/layouts.
```

Comment:
Let's not add a link to a page we know doesn't exist.  In fact, we should create these pages on backdropcms.org and just have place holder content there for now.  I'll set up https://backdropcms.org/layouts if you update this link :)